mappings/net/minecraft/loot/function/SetAttributesLootFunction.mapping

```diff
@@ -29,3 +35,24 @@ CLASS net/minecraft/class_137 net/minecraft/loot/function/SetAttributesLootFunct
 		METHOD method_615 serialize (Lcom/google/gson/JsonSerializationContext;)Lcom/google/gson/JsonObject;
 			ARG 1 context
 	CLASS class_139 Serializer
+	CLASS class_6155 Builder
+		FIELD field_31855 attributes Ljava/util/List;
+		METHOD method_35527 attribute (Lnet/minecraft/class_137$class_6156;)Lnet/minecraft/class_137$class_6155;
+			ARG 1 attribute
+	CLASS class_6156 AttributeBuilder
+		FIELD field_31856 name Ljava/lang/String;
+		FIELD field_31857 attribute Lnet/minecraft/class_1320;
+		FIELD field_31858 operation Lnet/minecraft/class_1322$class_1323;
+		FIELD field_31859 amountRange Lnet/minecraft/class_5658;
```

Comment:
Consider just using `amount` `durability` for `amountRange` `durabilityRange` if a number range is the way a number is accepted (assuming there is no literal string for these values)